### ISSUE TITLE
Textures not loading and exporting

### ISSUE THREAD

**Issue by alexfoo-productions** [mikeoverbay/TankExporter#55]
Hi, Textures are not loading and not exported for any tanks for client 1.6.0.3493.
![Screenshot 2022-04-07 155102](https://user-images.githubusercontent.com/1757157/162148329-3b223db3-331a-436d-b23f-886f648284c5.png)

**Comment by mikeoverbay** [comment]
Was it working before because it works fine on my machine?
If you have an AMD or Intel Video card, They have OpenGL issues.

**Comment by alexfoo-productions** [comment]
Thnx for the reply. Your answer gave me a hint to the problem. I'm running the program on a laptop with both Intel and discreet Nvidia GPU. Turns out the program was using the Intel GPU; so after I set the program to run using the discreet graphics through Nvidia Control Panel, the textures are showing correctly in the program now.

And as for the textures not exporting problem is solved also. Somehow the textures will not export to a desktop folder with  antivirus on (Avast) but will export happily to another drive (I have few hdds)

So all is running well, it was just from my side, maybe can include some info here in the readme so other's can be aware. With that tq.
